data/json/skills.json

```diff
@@ -69,7 +69,12 @@
         "ident" : "archery",
         "name" : "archery",
         "description" : "Your skill in using bow weapons, from hand-carved self bows to complex hunting crossbows.  Quiet and effective, they require strength of body and sight to wield, and are not terribly accurate over a long distance.",
```

Comment:
While you are here, could you please change the description, as archery no longer affects crossbows?